class-pinterest-for-woocommerce.php

```diff
@@ -835,6 +836,13 @@ public static function update_account_data() {
 					)
 				);
 
+				/*
+				 * For now we assume that the billing is not setup.
+				 * We will be able to check that only when the advertiser will be connected.
+				 * The billing is tied to advertiser.
+				 */
+				$data['is_billing_setup'] = false;
```

Comment:
This can be added as a default parameter on the `array_intersect_key` operation.

Reply:
You are right, the parameter is not present in the account data info. You can ignore my previous comment, sorry.